vcpkg/users/versioning-troubleshooting.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Troubleshoot versioning and feature selection issues
+description: Troubleshooting guide for common versioning issues
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 11/30/2023
+ms.prod: vcpkg
+ms.topic: troubleshooting-general
+---
+
+# Versioning troubleshooting guide
+
+This guide is intended for users experiencing issues with [versioning](./versioning.md).
+
+## Inspecting the version file for a port
```

Comment:
We really need a command that does this for the user, and it needs to work with registries.

Example:
```
./vcpkg x-package-info --versions --x-json fmt
{
  "versions": [
    {
      "git-tree": "dfe9aa860f5a8317f341a21d317be1cf44e89f18",
      "version": "10.1.1",
      "port-version": 0
    },
    {
      "git-tree": "d8209382ca192426861c6aa91d217ea8512bda46",
      "version": "10.1.0",
      "port-version": 0
    },
  //etc
}
```